packages/strapi-design-system/.storybook/preview.js

```diff
@@ -17,7 +17,6 @@ export const parameters = {
   actions: { argTypesRegex: '^on[A-Z].*' },
   darkMode: {
     // Override the default dark theme
-    current: 'light',
```

Comment:
Does it mean that the default displayed theme will be the dark one on storybook?

Reply:
here you can find the mode choice "hierarchy" applied by the [addon](https://storybook.js.org/addons/storybook-dark-mode) 
<img width="860" alt="image" src="https://user-images.githubusercontent.com/71838159/159722740-e61c8af8-0b1c-4852-a844-eec4ad3f3bb0.png">

the issue when we leave `current: 'light'` parameter is that everytime you go from one story to another or reload the page, it will set light as current.
with my modification:
- first time it will use your OS mode
- then the mode last used

WDYT @HichamELBSI ?

Reply:
Alright thanks ! Awesome